server/src/models/Task.ts

```diff
@@ -0,0 +1,13 @@
+class Task {
+  private timeStart: number;
+
+  constructor(public task: string) {
```

Comment:
nie powinnes zapisywac tutaj public jesli w konstruktorze przypisujesz normalnie wartosc do wlasciwosci obiektu. To slowo powinnes zapisac tylko jesli chcialbys linijke nizej pominac